scripts/dev/ci/travis_install.sh

```diff
@@ -111,18 +112,18 @@ tox --version
 case $TESTENV in
     py34-cov)
         pip_install codecov
-        apt_install xvfb $pyqt_pkgs libpython3.4-dev
+        apt_install xvfb "$pyqt_pkgs" libpython3.4-dev
         check_pyqt
         ;;
     pylint|vulture)
-        apt_install $pyqt_pkgs libpython3.4-dev
+        apt_install "$pyqt_pkgs" libpython3.4-dev
         check_pyqt
         ;;
     flake8)
         apt_install libpython3.4-dev
         ;;
     docs)
-        apt_install $pyqt_pkgs asciidoc libpython3.4-dev
+        apt_install "$pyqt_pkgs" asciidoc libpython3.4-dev
```

Comment:
It doesn't matter if it's unquoted - not because of that, but because there are no spaces in the package name. `apt_install` quoting things doesn't help if they arrive split in the first place :wink:

However, as-is this is wrong and [fails](https://travis-ci.org/The-Compiler/qutebrowser/jobs/148124899#L242-L243) because it tries to install the whole thing as a single package.